File: Java Programs/Practice/PalindromeNumberCheck.java
import java.util.Scanner;

public class PalindromeNumberCheck
{
	public static void main(String args[])
	{
		Scanner sc = new Scanner(System.in);

		System.out.print("\n Enter a number : ");
		int n = sc.nextInt();

		int m = n;
		int rev = 0;

		while(m != 0)
		{
			int rem = m % 10;
			rev = rev * 10 + rem;
			m = m / 10;
		}

		if(n == rev)
		{
			System.out.println("\n " + n + " is a palindrome");
		}
		else
		{
			System.out.println("\n " + n + " is not a palindrome");
		}
		
		sc.close();
	}
}	

File: Python Programs/palindrome_number_check.py
n = int(input("Enter a number : "))

m = n
rev = 0

while(m != 0):
    rem = m % 10
    rev = rev * 10 + rem
    m = m // 10

print("The reversed number is : ", rev)

if(n == rev):
    print(n, " is a palindrome number")
else:
    print(n, " is not a palindrome number")
